### PR TITLE
Assault !st tiers, assault global, appraisal global

### PR DESCRIPTION
**Pull request by KnowOne134**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](https://github.com/project-topaz/topaz/blob/master/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/zones/Aht_Urhgan_Whitegate/npcs/Rytaal.lua [hunk 1]
```diff
@@ -1,93 +1,106 @@
...
+        player:setCharVar("Assault_Armband", 0)
+        for _, v in pairs(assaultOrders) do
+            if player:hasKeyItem(v.KI) then
+                player:delKeyItem(v.KI)
+            end
         end
-        player:setCharVar("AssaultComplete",0)
     elseif player:getCurrentMission(TOAU) > tpz.mission.id.toau.PRESIDENT_SALAHEEM or (player:getCurrentMission(TOAU) == tpz.mission.id.toau.PRESIDENT_SALAHEEM and player:getCharVar("AhtUrganStatus") >= 1) then
+        local lastTagTime = lastIDtag - 1009897200
+        local diffday =  math.floor((os.time() - lastIDtag)/86400)
```
Comment (zach2good) [inline scripts/zones/Aht_Urhgan_Whitegate/npcs/Rytaal.lua]: You could support the ROV KI here:
```
local mustWaitTime = 86400 -- 1 day in seconds
if player:hasKeyItem(tpz.ki.RHAPSODY_IN_AZURE) then mustWaitTime = 600 end -- set to 10 minutes
local diffday =  math.floor((os.time() - lastIDtag)/mustWaitTime)
```

Comment (KnowOne134) [reply scripts/zones/Aht_Urhgan_Whitegate/npcs/Rytaal.lua]: Yes there will be alot of changes to retail today configuration. I wrote this all for in era. I did add 1 settings so for for ENABLE_TOAU_ERA and set it for how many people can enter an assault. Era is 3 to 6. Retail is 1 to 6. But most of these changes will be done once I get it setup and working

**Comment by KnowOne134** [comment]
2 areas are tested and functioning now, L. Cleansing and Imperial Agent Rescue. Rytaal function, runic portal use, appraisals, points, temp items, 100 points on fail attempts, level capping fights, all restrictions to enter a assault with a setting for ERA or retail entering. Era = 3 to 6 player retail 1 to 6

**Comment by KnowOne134** [comment]
I'm gonna close this until I can finish it and ready for review. Life has got its claws into me atm and I had to step back from alot of things.

**Comment by cocosolos** [comment]
Sad to hear that. Hope everything works out for you, looking forward to eventual release. Good luck!

**Pull request closed without merging**